sites/diverseeducation.com/server/public/robots.txt

```diff
@@ -1,9 +0,0 @@
-# www.robotstxt.org/
-# www.google.com/support/webmasters/bin/answer.py?hl=en&answer=156449
-
-User-agent: *
-Disallow: /__
-Disallow: /print/content
-
-Sitemap: https://www.diverseeducation.com/sitemap.xml
-Sitemap: https://www.diverseeducation.com/sitemap-google-news.xml
```

Comment:
Were you told to remove this?

Believe this needs to be restored and the sitemap URLs adjusted as necessary.

Reply:
@Shinsina Oops.. nope